scripts/generate_assets.py

```diff
@@ -0,0 +1,101 @@
+from birdclef.models.embedding.tilenet import TileNet
+import pandas as pd
+from birdclef.workflows import motif
+from pathlib import Path
+import IPython.display as ipd
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from sklearn.decomposition import PCA
+from sklearn.cluster import KMeans
+from sklearn.preprocessing import LabelEncoder
+from jinja2 import Environment, FileSystemLoader
+import webbrowser
+import click
+from ast import literal_eval
+import os
+
+
+@click.command()
+@click.option('--intra', default="brnowl", help='Singular species to perform intraclass inspection')
+@click.option('--inter', default=["brnowl", "skylar", "norcar"], help='Array of 3 species to perform interclass comparison')
+def main(intra, inter):
+    model = TileNet.load_from_checkpoint(
+        "../data/processed/model/2022-04-12-v4/"
```

Comment:
The checkpoint should be a click option -- check out the classify predict options.